src/language-html/printer-html.js

```diff
@@ -30,6 +30,7 @@ const {
   getPrettierIgnoreAttributeCommentData,
   hasPrettierIgnore,
   inferScriptParser,
+  isCustomBlock,
```

Comment:
Please rename it to `isVueCustomBlock`. This parser and printer are also used for HTML and Angular, so Vue-specific things should be marked as such.